CMakeLists.txt

```diff
@@ -105,7 +107,7 @@ if(OPENMP)
 endif()
 find_package(NetCDF REQUIRED C Fortran)
 find_package(ESMF MODULE REQUIRED)
-if(S2S)
+if(S2S OR CDEPS)
```

Comment:
It think this is not generic enough. If you try to create configuration without S2S and CDEPS like FV3+HYCOM under HAFS this will fail beacuse this configuration still needs CMEPS and PIO support. I think we need to set CMEPS explicitly like we did in other components and get rid of shortcuts like S2S. This is already done in HAFs side. There is an active issue - https://github.com/ufs-community/ufs-weather-model/issues/416 - related with the creating more flexible build system under UFS.